data_model.md

```diff
@@ -226,10 +229,37 @@ for more detailed explanation of forecast parameters):
 Additional parameters, such as model configuration parameters, may be
 specified under the “extra parameters” key.
 
-
 Each Forecast has data associated with it. The format of this data is
 found in the Data Format section below
 
+#### Probabilistic forecasts
+{: .anchor}
+
+The Solar Forecast Arbiter supports the specification of a full
```

Comment:
`supports specifying a probabilistic forecast by a cumulative distribution function.`

`The metadata for a probabilistic forecast is the same as for a [deterministic forecast](#single-valued-forecasts) `